src/project/git/GitCommandGitProject.ts

```diff
@@ -48,11 +53,15 @@ export class GitCommandGitProject extends NodeFsLocalProject implements GitProje
      * @param {RepoRef} id
      * @param {string} baseDir
      * @param {ProjectOperationCredentials} credentials
+     * @param release call this when you're done with the project. make its filesystem resources available to others.
```

Comment:
the release function is added so that you can let go of the resource. Most places default it to a no-op

Reply:
LGTM